src/renderer/api/endpoints/crd.api.ts

```diff
@@ -87,7 +102,7 @@ export class CustomResourceDefinition extends KubeObject {
   }
 
   getVersion() {
-    return this.spec.version;
+    return this.spec.version ?? this.spec.versions.find(a => a.served)?.name;
```

Comment:
actually I am not sure on this logic here - while it does work I just spotted that I am using served here and storage above. I just found a CRD that has multiple versions - Elastic Search
http://localhost:8001/apis/apiextensions.k8s.io/v1/customresourcedefinitions/elasticsearches.elasticsearch.k8s.elastic.co

Both the first and the `storage: true` version are v1, which is logically the correct value. Their `v1beta` has `served: true` and `storage: false`